app/features-json/project_json_controller.rb

```diff
@@ -33,8 +33,9 @@ class ProjectJSONController < APIController
       end
 
       repo_config = GitHubRepoConfig.from_octokit_repo!(repo: selected_repo)
-
-      platform, lane = params["lane"].split(" ") # Split "ios test_lane"
+      lane = params["lane"]
+      # If platform is empty string, means no platform
+      platform = params["platform"] == "" ? :no_platform : params["platform"]
```

Comment:
won't this be the same error since the backend will be getting `no_platform` as a platform name?

Reply:
You are right. The issue is not while configuring the project, but during the build process. https://github.com/fastlane/ci/pull/951 automatically fixed the issue for manual builds.